docs/intro.rst

```diff
@@ -22,3 +22,5 @@ partitioning program. Anaconda provides advanced debugging features such as
 remote logging, access to the python interactive debugger, and remote saving of
 exception dumps.
 
+For more news about Anaconda development and planned features you can follow this
+`blog site <https://rhinstaller.wordpress.com>`_.
```

Comment:
I'd go with: s/this/our/ ; s/site// ; and move the "our" into link.

Reply:
Fixed.